build.cake

```diff
@@ -341,13 +347,35 @@ Task("PackageFramework")
         {
             Version = packageVersion,
             BasePath = CurrentImageDir,
-            OutputDirectory = PACKAGE_DIR
+            OutputDirectory = PACKAGE_DIR,
+            Symbols = true
         };
 
         NuGetPack("nuget/framework/nunit.nuspec", settings);
         NuGetPack("nuget/nunitlite/nunitlite.nuspec", settings);
     });
 
+Task("PackageFrameworkSnupkg")
+    .Description("Creates NuGet snupkg source packages of the framework")
+    .IsDependentOn("CreateImage")
+    .Does(() =>
+    {
+        CreateDirectory(PACKAGE_DIR);
+
+        var settings = new NuGetPackSettings
+        {
+            Version = packageVersion,
+            BasePath = CurrentImageDir,
+            OutputDirectory = PACKAGE_DIR,
+            Symbols = true,
+            // snupkg is not yet supported by Cake, https://github.com/cake-build/cake/issues/2362
+            ArgumentCustomization = args => args.Append("-SymbolPackageFormat snupkg")
+        };
+
+        NuGetPack("nuget/framework/nunit.snupkg.nuspec", settings);
+        NuGetPack("nuget/nunitlite/nunitlite.snupkg.nuspec", settings);
+    });
```

Comment:
Too bad the same .nuspec files can't be reused and a single `nuget pack` command be run. I don't know enough about it yet to know if that's attainable when using a nuspec.

Reply:
It may be possible. That was my first approach and I got errors, so I just went with this. If we can clean it up later I'd be cool, but I just want to get the release out. I'd hoped to have it done today.